app/src/main/java/com/mapbox/maps/testapp/examples/GesturesActivity.kt

```diff
@@ -41,6 +42,10 @@ class GesturesActivity : AppCompatActivity() {
   private var focalPointLatLng: Point? = null
   private var pointAnnotationManager: PointAnnotationManager? = null
   private lateinit var binding: ActivityGesturesBinding
+  private lateinit var rotateListener: OnRotateListener
```

Comment:
Actually, I think `gestureAlertsAdapter` doesn't need to be `lateinit var`, it could be `val` and initialized already at definition:
```Kotlin
private val gestureAlertsAdapter = GestureAlertsAdapter()
```
Then, these `*Listener`s could also be just `val` instead of `lateinit var`.

Wdyt? Would it be ok to do that change here or shall we do it in another PR?

Reply:
I'm okay with making the change here.